include/onnc/Analysis/GlobalStatistics.h

```diff
@@ -0,0 +1,30 @@
+//===- SkyGlobal.h --------------------------------------------------------===//
+//
+//                               Skymizer
+//
+// Copyright (C) 2013, Skymizer Inc.. All rights reserved.
+//
+//===----------------------------------------------------------------------===//
+#ifndef ONNC_ANALYSIS_GLOBALSTATISTICS_H
+#define ONNC_ANALYSIS_GLOBALSTATISTICS_H
+#include <onnc/Analysis/Statistics.h>
+#include <onnc/Analysis/StatisticsGroup.h>
+
+namespace onnc {
+
+namespace global {
+
+Statistics* GlobalStat();
```

Comment:
Use `stats()` instead of GlobalStat(). Because we already have `global` namespace, the Global prefix is redundant.

Reply:
Modify API name @7423059ae